schema/bom-1.7.xsd

```diff
@@ -390,6 +396,57 @@ limitations under the License.
         </xs:anyAttribute>
     </xs:complexType>
 
+    <xs:simpleType name="tlpType" default="CLEAR">
```

Comment:
default is not allowed here, it needs to be moved to the element where the type is applied

Reply:
done via 98d888da511f283331888c7f91a90dcc24cbe38a